src/appmixer/notion/core/MakeApiCall/MakeApiCall.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+const API_VERSION = '2022-06-28';
+
+module.exports = {
+    async receive(context) {
+        const { url, method, body } = context.messages.in.content;
+
+        const requestOptions = {
+            method: method,
+            url: url,
+            headers: {
+                'Authorization': `Bearer ${context.auth.accessToken}`,
+                'Content-Type': 'application/json',
+                'Notion-Version': API_VERSION
```

Comment:
another place with the API_VERSION. btw can't we use the `lib.callEndpoint` here ?